test/integration/next-image-new/default/pages/invalid-Infinity-width.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Image from 'next/image'
+
+export default function Page() {
```

Comment:
You’ll need to add this same file to app-dir/app/invalid-infinity/page.js

Reply:
done, thank you for your reminder.